adaptation.rb

```diff
@@ -64,15 +64,12 @@ def usage(s)
     next if u.flags2.killed
     u.status.misc_traits.each { |t|
         if t.id == :CaveAdapt
-            # TBD: expose the color_ostream console and color values of
-            # t.value based on adaptation level
             if mode == 'show'
                 if df.respond_to?(:print_color)
-                    print "Unit #{u.id} (#{u.name}) has an adaptation of "
+                    df.print_color(COLOR_WHITE, "Unit #{u.id} (#{u.name}) has an adaptation of ")
```

Comment:
This should use COLOR_RESET instead, which you'll probably need to define as `-1` above. White and black should be avoided because they are the default background colors of some terminals (e.g. the default color scheme of macOS's Terminal.app has a white background).

Reply:
I actually thought about using COLOR_RESET... I forgot to do it along the way... *facepalm

Reply:
By the way, looking at the [implementation](https://github.com/DFHack/dfhack/blob/fb44b26b47eca729646e316efc22cfe2a138691e/plugins/ruby/ruby.cpp#L575-L585), it appears that print_color does already reset the color - but I'm fine with explicitly resetting it for clarity between other print_color calls.